added meta html values (#3213)
plone/volto: docs/source/deploying/simple.md

---
html_meta:

>   "description": "Volto is a Node application that runs on your machine/server and listens to a port. Once you are ready to deploy it, you should build it running:"

Reviewer: ```suggestion
  "description": "Simple deployment of a Volto application"
```

>   "property=og:description": "Volto is a Node application that runs on your machine/server and listens to a port. Once you are ready to deploy it, you should build it running:"

Reviewer: ```suggestion
  "property=og:description": "Simple deployment of a Volto application"
```
